cmd/namespace/create.go

```diff
@@ -60,10 +60,11 @@ func Create(ctx context.Context) *cobra.Command {
 			analytics.TrackCreateNamespace(err == nil)
 			return err
 		},
-		Args: utils.ExactArgsAccepted(1, ""),
+		Args: utils.MaximumNArgsAccepted(2, ""),
 	}
 
 	options.Members = cmd.Flags().StringArrayP("members", "m", []string{}, "members of the namespace, it can the username or email")
+	cmd.Flags().BoolVarP(&options.SetCurrentNs, "use", "", false, "use the newly created namespace as the current namespace")
```

Comment:
using `false` as the default value change the behavior we have until now. Should we use as default value `true` in order to keep the same behavior when the input is the same as before? cc @jLopezbarb

Reply:
Could you specify what do you mean by 'behaviour' here? 

Earlier the default value was set as `true` but then, new ns was selected as current ns with flag being passed and not being passed. After some try, I changed it to `false` and it's running well as we expect it now.

Reply:
I mean, if the user executes `okteto ns create my-namespace` then the namespace will not be changed because by default `--use` opt is false. All users using this command would be affected since now by default the namespace will not be switched following the changes of this PR. I think we could set `--use` to true to keep the same behavior if the user does not set the `--use` flag in the command. Is this the behavior we want? cc @jLopezbarb

Reply:
Oh! I thought we want to make it work something like I mentioned earlier. If the behavior is the same as you're saying, this PR would have merged earlier because I was thinking of implement different behavior rather than you mentioned. 

Th behaviour you mentioned was already working. I'll push the new changes now. Thanks for confirming!

Reply:
Fixed!